docs/docs/rest-api/public/api/v2/leader.raml

```diff
@@ -50,14 +50,18 @@ delete:
       required: false
       description: URI to restore zk state.
       example: file:///var/backup/marathon/backup
+    delay?:
```

Comment:
wouldn't seconds be better for this?

Also I would go with delaySeconds instead of delay. Looking at HealthCheck raml definition, this is how we named properties in the past (and from the user perspective it is actually convenient too, because you need to know the unit anyway)

Also shouldn't the default be 0 to be backward compatible?

Reply:
The value for 1.5 was 500ms

https://github.com/mesosphere/marathon/blob/releases/1.5/src/main/scala/mesosphere/marathon/core/election/impl/CuratorElectionService.scala#L50

After some consideration, I decided that modifying the API was unnecessary complexity. So, I just reverted that part.